src/plugins/notebook/components/NotebookEntry.vue

```diff
@@ -272,16 +276,21 @@ export default {
                 return;
             }
 
-            const entryPos = this.entryPosById(entryId);
+            target.blur();
+
             const value = target.textContent.trim();
-            if (this.currentEntryValue !== value) {
-                target.textContent = value;
+            if (this.currentEntryValue === value) {
+                return;
+            }
 
-                const entries = getNotebookEntries(this.domainObject, this.selectedSection, this.selectedPage);
-                entries[entryPos].text = value;
+            target.textContent = value;
+            this.currentEntryValue = value;
 
-                this.updateEntries(entries);
-            }
+            const entries = getNotebookEntries(this.domainObject, this.selectedSection, this.selectedPage);
+            const entryPos = this.entryPosById(entryId);
+            entries[entryPos].text = value;
+
+            this.updateEntries(entries);
```

Comment:
This seems wrong to me architecturally. A notebook entry should not have access to all the entries in the notebook. Instead we should emit an update event to the notebook, and let the notebook find the correct entry (via id) and update it. so the flow of information stays consistent.

Reply:
Agree, need to change this.